mkdocs.yml

```diff
@@ -56,6 +56,10 @@ nav:
       - Using an AWS AMI: guides/installation-ami.md
       - Using Helm: guides/installation-helm.md
       - Tanzu Community Edition: guides/installation-tce.md
+    - Observability:
+      - '': guides/observability.md
+      - Structured logging: guides/observability-logging.md
```

Comment:
@antoineco no such guide exists in the repo. is it possible you failed to commit it or something?

Reply:
It was meant as a placeholder because I thought maybe someone would write that one in parallel. We should probably comment this.